meta/Meta.cpp

```diff
@@ -1856,6 +1856,37 @@ sai_status_t Meta::objectTypeGetAvailability(
     return status;
 }
 
+sai_status_t Meta::queryAattributeCapability(
+        _In_ sai_object_id_t switchId,
+        _In_ sai_object_type_t objectType,
+        _In_ sai_attr_id_t attrId,
+        _Out_ sai_attr_capability_t *Capability)
```

Comment:
minor, Capability -> capability

Reply:
Renamed to use lower case for variable "capability" and renamed the function name to remove the extra "a" all across files that referenced this function name.